### PR TITLE
Fix deprecated set-output

### PR DESCRIPTION
**Pull request by IAlibay**

### CODE REVIEW

**Comment by IAlibay** [comment]
cycling PR for CI

File: .github/workflows/gh-ci.yaml [hunk 1]
```diff
@@ -26,21 +26,28 @@ defaults:
...
+  MDANALYSIS_PREVIOUS_RELEASE: "2.3.0"
 
 jobs:
   main-tests:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         os: [ubuntu-latest]
-        python-version: ["3.8", "3.9", "3.10"]
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
```
Comment (lilyminium) [inline .github/workflows/gh-ci.yaml]:
```suggestion
        python-version: ["3.9", "3.10", "3.11"]
```

Probably can remove from test matrix?

**Review by lilyminium — APPROVED** [review]
LGTM -- thanks for picking this up. Just a minor non-blockign comment.

File: .github/workflows/gh-ci.yaml (hunk 1 above)
Comment (IAlibay) [reply .github/workflows/gh-ci.yaml]: looks to me like we should be using our new action 🙃

**Pull request merged**